image/image_decoder.h

```diff
@@ -94,20 +95,9 @@ class ImageDecoder {
 	 * until destroy() or loadStream() is called, or until the destructor of
 	 * this ImageDecoder is called.
 	 *
-	 * The format of the palette is the same as that of the PaletteManager's palette.
-	 * (interleaved RGB values).
-	 *
-	 * @return The decoded palette, or undefined if no palette is present.
+	 * @return The decoded palette, or empty if no palette is present.
 	 */
-	virtual const byte *getPalette() const { return 0; }
-
-	/**
-	 * Query whether the decoded image has a palette.
-	 */
-	virtual bool hasPalette() const { return getPaletteColorCount() != 0; }
```

Comment:
Would it be possible to keep this as an inlined wrapper around `getPalette().empty()`?

Reply:
Sure, I can add that back. Should I also swap the uses back as well?

Reply:
> Sure, I can add that back. Should I also swap the uses back as well?

Yes please :)

Reply:
Added hasPalette back and included the palette header